daktari/config.py

```diff
@@ -105,6 +107,33 @@ def parse_raw_config(config_path: Path, raw_config: str) -> Optional[Config]:
     return Config(min_version, title, checks)
 
 
+def is_python_version_on_path() -> bool:
+    path = (os.getenv("PATH") or "").lower()
+    try:
+        python3_version_raw = run_command(["python3", "--version"]).stdout.rstrip("\n")
+        python3_version = re.search("3\\.\\d+\\.\\d+", python3_version_raw)
```

Comment:
Could use a raw string here to eliminate a few backslashes:

```suggestion
        python3_version = re.search(r"3\.\d+\.\d+", python3_version_raw)
```